proposals/Exceptions.md

```diff
@@ -240,19 +234,21 @@ If the `kind` is `Exception`:
 
 | Field | Type | Description |
 |-------|------|-------------|
-| type  | `varuint32` | exception index to table in the *exception type section* |
+| tag   | varuint32 | index into exception section |
```

Comment:
Sorry if my comment wasn't clear, I was referring to the `tag` field specifically, which I can't make sense of. What would it be used for? A tag is the very thing that gets imported, so why a reference to another tag definition here? (Consider, for example, that you of course ought to be able to import exceptions without defining any locally -- that can't possibly work if the import somehow needs to refer to a local definition.)